### ISSUE TITLE
2023.2beta2, Strings of window that appears first time you open add-on store are in English and missing from .po translation

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#15272]
### Actual behavior:

Check italian nvda.po file, mentioned string is not present. Also, you can use NVDA 2023.2beta2, open add-on store for the first time, you will find that the warning message is all in english.

### Expected behavior:

I should find that string on my nvda.po file in order to translate it.
String is:
Add-ons are created by the NVDA community and are not vetted by NV Access. NV Access cannot be held
responsible for add-on behavior. The functionality of add-ons is unrestricted and can include accessing your
personal data or even the entire system. 

Test made with nvda 2023.2beta2 portable, and obviously updating translation system

**Comment by seanbudd** [comment]
The translation string is yet to appear, #15246  was merged after friday's string push from NVDA to the translation system, not before
